modules/cdc-ext/src/main/java/org/apache/ignite/cdc/conflictresolve/CacheConflictResolutionManagerImpl.java

```diff
@@ -38,22 +39,42 @@
      */
     private final String conflictResolveField;
 
+    /** CLuster Id. */
+    private final byte clusterId;
+
     /** Grid cache context. */
     private GridCacheContext<K, V> cctx;
 
     /**
      * @param conflictResolveField Field to resolve conflicts.
      */
-    public CacheConflictResolutionManagerImpl(String conflictResolveField) {
+    public CacheConflictResolutionManagerImpl(String conflictResolveField, byte clusterId) {
         this.conflictResolveField = conflictResolveField;
+        this.clusterId = clusterId;
     }
 
     /** {@inheritDoc} */
     @Override public CacheVersionConflictResolver conflictResolver() {
+        cctx.logger(this.getClass()).info("Conflict resolver created [" +
+            "cache=" + cctx.name() +
+            ", clusterId=" +  clusterId +
+            ", conflictResolveField=" + conflictResolveField + ']'
+        );
+
+        IgniteLogger log = cctx.logger(CacheVersionConflictResolverImpl.class);
```

Comment:
cctx.logger() creates new logger instance each time it is called. Do we need it? May be it is better to save it once in the dedicated variable?

Reply:
Fixed.